library/src/main/java/com/bumptech/glide/load/resource/bitmap/CenterCrop.java

```diff
@@ -13,24 +13,16 @@
  *
  * Does not maintain the image's aspect ratio
  */
-public class CenterCrop extends BitmapTransformation {
+public class CenterCrop extends PositionedCrop {
   private static final String ID = "com.bumptech.glide.load.resource.bitmap.CenterCrop";
   private static final byte[] ID_BYTES = ID.getBytes(CHARSET);
 
   public CenterCrop(Context context) {
-    super(context);
+    super(context, 0.5f, 0.5f);
```

Comment:
Hmm, I think I figured out where you got start/end from: fitStart/fitEnd (`Matrix.ScaleToFit`).